zerver/openapi/zulip.yaml

```diff
@@ -20548,6 +20563,19 @@ paths:
                         [mentions]: /help/mention-a-user-or-group
                     - $ref: "#/components/schemas/GroupSettingValue"
                   example: 11
+                can_remove_members_group:
+                  allOf:
+                    - description: |
+                        A [group-setting value][setting-values] defining the set of users who
+                        have permission to remove members from this user group.
```

Comment:
Do we also want to mention here that user with permission to manage the group can remove members? We do not do this for other settings, so have not added that here as well.

Reply:
Also, for `can_join_group` and `can_leave_group` settings we can mention that user who has permission to add and remove members can join and leave the group.

Reply:
I think probably these details are worth spelling out somewhere, but I'm not sure if this is the right place for it, or some sort of more general documentation on the groups model. Can you post in #documentation about where it would be best to explain these permissions details? I feel like it might be kinda covered in the /help/ articles.

Reply:
https://chat.zulip.org/#narrow/channel/19-documentation/topic/user.20groups/near/1992714